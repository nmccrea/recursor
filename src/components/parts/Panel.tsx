import styled from "styled-components"
import { color, border, shadow } from "styles/utilities"

export const Panel = styled.div`
  padding: 1rem;
  background-color: ${color.transparent.white1};
  ${border.borderRadius}
  ${shadow.small}
`
