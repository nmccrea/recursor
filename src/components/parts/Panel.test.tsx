import { render } from "@testing-library/react"
import "jest-styled-components"
import React from "react"
import { Panel } from "./Panel"

describe("`<Panel />`", () => {
  it("renders correctly", () => {
    const { container } = render(<Panel>TEST CONTENT</Panel>)

    expect(container).toMatchSnapshot()
  })
})
