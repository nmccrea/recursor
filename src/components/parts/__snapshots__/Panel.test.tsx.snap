// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`<Panel />\` renders correctly 1`] = `
.c0 {
  padding: 1rem;
  background-color: rgba(255,255,255,0.1);
  border-radius: 0.5rem;
  box-shadow: 0px 3px 1px -2px rgba(0,0,0,0.2), 0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12);
}

<div>
  <div
    class="c0"
  >
    TEST CONTENT
  </div>
</div>
`;
